### DISCUSSION TITLE
Any plan to release a new version?

### DISCUSSION THREAD

**Discussion by blaisewang** [jimporter/mike#180] [General]
Mike has been a very useful tool for us. No releases have been made after October 4, 2021. However, the project has been in active development since then. I would like to know if there are any plans or milestones for releasing a new version?

Also, if I understand correctly, the `docs` folder is not necessary in some cases, such as the `mike delete` command. But now I need to create an empty `docs` folder to use the delete command.

**Comment by jimporter** [comment]
Eventually, yes. However, there are still a few issues in the [2.0 milestone](https://github.com/jimporter/mike/milestone/1) that need wrapped up. The highest priority of these is #106, which is a pretty hard blocker (the others aren't quite blockers, but very nice to have).

If you'd like to see 2.0 released sooner, please try to help resolve the issues in the milestone. Unfortunately, I don't have a ton of time to work on this project at the moment, so it might take a bit before I can get to the issues myself.
